src/server/middleware/readonly/readonly.go

```diff
@@ -67,9 +67,11 @@ func MiddlewareWithConfig(config Config, skippers ...middleware.Skipper) func(ht
 
 	return middleware.New(func(w http.ResponseWriter, r *http.Request, next http.Handler) {
 		if config.ReadOnly(r) {
-			pkgE := internal_errors.New(nil).WithCode(internal_errors.DENIED).WithMessage("The system is in read only mode. Any modification is prohibited.")
-			serror.SendError(w, pkgE)
-			return
+			if r.Method == http.MethodDelete || r.Method == http.MethodPost || r.Method == http.MethodPatch || r.Method == http.MethodPut {
+				pkgE := internal_errors.New(nil).WithCode(internal_errors.DENIED).WithMessage("The system is in read only mode. Any modification is prohibited.")
+				serror.SendError(w, pkgE)
+				return
+			}
```

Comment:
These changes are not needed here, only need to join the custom's skipers and the safeMethodSkipper .

Reply:
thanks, updated.